### DISCUSSION TITLE
Contribution card not showing the correct stats for my profile

### DISCUSSION THREAD

**Discussion by samyakagarwal20** [anuraghazra/github-readme-stats#3091] [Q&A] [answered]
### Describe the bug

Hi,

I am hosting the fork of this repo on my personal vercel account. I am using the following link to fetch the github stats for my profile

https://github-readme-stats-samyakagarwal20.vercel.app/api?username=samyakagarwal20&theme=github_dark

The above link fetches the following data for my profile
![Samyak's GitHub stats](https://github-readme-stats-samyakagarwal20.vercel.app/api?username=samyakagarwal20&theme=github_dark)

but, to my surprise, the stats are not getting loaded correctly as per my contribution graph displayed below:
![image](https://github.com/anuraghazra/github-readme-stats/assets/40698105/0aeafa02-013a-4704-9889-e8488897a931)

Please help me out to resolve these issues.

**Comment by rickstaa** [comment, marked as answer]
@samyakagarwal20 thanks for your question. You are confusing contributions with commits (see https://github.com/anuraghazra/github-readme-stats/discussions/1770#number-of-commits-is-incorrect). We currently do not show contributions.

**Reply by samyakagarwal20** [reply]
Got it. Thanks for the reference though.
